pkgs/servers/jigasi/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, dpkg, jdk11, nixosTests }:
+
+let
+  pname = "jigasi";
+  version = "1.1-231-gd792d0f";
+  arch = "amd64";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_${arch}.deb";
+    sha256 = "0riij1i3k33rgzs4gqjffvcrx18a1dmz825nzk85xpsh972z0a4n";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "jigasi";
  version = "1.1-231-gd792d0f";
  
  src = fetchurl {
    url = "https://download.jitsi.org/stable/${pname}_${version}-1_amd64.deb";
    sha256 = "0riij1i3k33rgzs4gqjffvcrx18a1dmz825nzk85xpsh972z0a4n";
  };
```
or can we easily build from source?